application/controllers/survey_response.py

```diff
@@ -6,82 +6,37 @@
     convert_file_object_to_string_base64
 from application.controllers.json_encrypter import Encrypter
 from application.controllers.rabbitmq_submitter import RabbitMQSubmitter
-from application.controllers.service_request import service_request
 from flask import current_app
 from structlog import get_logger
-from werkzeug.utils import secure_filename
+from application.controllers.service_helper import get_case_group, get_collection_exercise, get_survey_ref
 
 log = get_logger()
 
-
-FILE_EXTENSION_ERROR = 'The spreadsheet must be in .xls ot .xlsx format'
+FILE_EXTENSION_ERROR = 'The spreadsheet must be in .xls or .xlsx format'
 FILE_NAME_LENGTH_ERROR = 'The file name of your spreadsheet must be less than 50 characters long'
-UPLOAD_SUCCESSFUL = 'Upload successful'
-UPLOAD_UNSUCCESSFUL = 'Upload failed'
-INVALID_UPLOAD = 'The upload must have valid case_id and a file attached'
-NOT_FOUND_SERVICE_RESPONSE = 'A request to an external service returned no data'
 QUEUE_NAME = 'Seft.Responses'
 
 
 class SurveyResponse(object):
```

Comment:
This probably doesn't need to be a class and could just be a few functions

Reply:
Yep, but maybe one for the future. Just need to get this PR in before it disintegrates. Its been alive far to long